### ISSUE TITLE
Error at executing command: pex user Ask4 group set Owner

### ISSUE THREAD

**Issue by Ask4Lp** [PEXPlugins/PermissionsEx#2301]
### Description ###

I tried to set my own minecraft account the Owner, but there was an error. the server runs on my local pc and I use the ip localhost, it would be nice if someone could help me

### What PEX Saw ###

**Reason:** While CONSOLE was executing /pex

**Stacktrace:**
```
java.lang.RuntimeException: java.lang.RuntimeException: Default user group is not defined. Please select one using the "default: true" property
	at ru.tehkode.permissions.commands.CommandsManager.execute(CommandsManager.java:121)
	at ru.tehkode.permissions.bukkit.PermissionsEx.onCommand(PermissionsEx.java:159)
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:44)
	at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:141)
	at org.bukkit.craftbukkit.v1_8_R3.CraftServer.dispatchCommand(CraftServer.java:642)
	at org.bukkit.craftbukkit.v1_8_R3.CraftServer.dispatchServerCommand(CraftServer.java:628)
	at net.minecraft.server.v1_8_R3.DedicatedServer.aO(DedicatedServer.java:404)
	at net.minecraft.server.v1_8_R3.DedicatedServer.B(DedicatedServer.java:368)
	at net.minecraft.server.v1_8_R3.MinecraftServer.A(MinecraftServer.java:657)
	at net.minecraft.server.v1_8_R3.MinecraftServer.run(MinecraftServer.java:560)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.RuntimeException: Default user group is not defined. Please select one using the "default: true" property
	at ru.tehkode.permissions.backends.FileBackend.getDefaultGroup(FileBackend.java:143)
	at ru.tehkode.permissions.PermissionManager.getDefaultGroup(PermissionManager.java:291)
	at ru.tehkode.permissions.PermissionManager.getDefaultGroup(PermissionManager.java:280)
	at ru.tehkode.permissions.PermissionUser.getGroups(PermissionUser.java:219)
	at ru.tehkode.permissions.PermissionUser.getOption(PermissionUser.java:130)
	at ru.tehkode.permissions.PermissionEntity.getOptionBoolean(PermissionEntity.java:294)
	at ru.tehkode.permissions.PermissionEntity.initialize(PermissionEntity.java:53)
	at ru.tehkode.permissions.PermissionUser.initialize(PermissionUser.java:51)
	at ru.tehkode.permissions.ProxyPermissionUser.initialize(ProxyPermissionUser.java:42)
	at ru.tehkode.permissions.PermissionManager.getUser(PermissionManager.java:118)
	at ru.tehkode.permissions.backends.FileBackend.getRegisteredUsers(FileBackend.java:200)
	at ru.tehkode.permissions.PermissionBackend.getUsers(PermissionBackend.java:203)
	at ru.tehkode.permissions.PermissionManager.getUsers(PermissionManager.java:144)
	at ru.tehkode.permissions.bukkit.commands.PermissionsCommand.autoCompletePlayerName(PermissionsCommand.java:106)
	at ru.tehkode.permissions.bukkit.commands.PermissionsCommand.autoCompletePlayerName(PermissionsCommand.java:73)
	at ru.tehkode.permissions.bukkit.commands.UserCommands.userSetGroup(UserCommands.java:520)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at ru.tehkode.permissions.commands.CommandsManager$CommandBinding.call(CommandsManager.java:252)
	at ru.tehkode.permissions.commands.CommandsManager.execute(CommandsManager.java:114)
	... 10 more
```
### PEX configuration ###
```yaml
multiserver:
  use-netevents: true
permissions:
  debug: false
  allowOps: false
  user-add-groups-last: true
  log-players: false
  createUserRecords: false
  backend: file
  informplayers:
    changes: false
  basedir: plugins/PermissionsEx
  backends:
    file:
      type: file
      file: permissions.yml
  superperms: {}
updater: true
alwaysUpdate: false
```
### Permissions database ###
```yaml
groups:
  Member:
    options:
      default: true
    permissions:
  VIP:
    options:
      default: false
    permissions:
  VIP+:
    options:
      default: false
    permissions:
  MVP:
    options:
      default: false
    permissions:
  MVP+:
    options:
      default: false
    permissions:
  Admin:
    options:
      default: false
    permissions:
    - modifyworld.*
  Owner:
    options:
      default: false
    permissions:
    - modifyworld.*
schema-version: 1
users:
  e260e751-b636-3e82-ae6a-ee9792bc7311:
    options:
      name: Ask4
```

**Comment by Ask4Lp** [comment]
I tried to set my own minecraft account the Owner, but there was an error. the server runs on my local pc and I use the ip localhost, it would be nice if someone could help me

**Comment by CzEMC** [comment]
Try to Set your default group by typing "/pex set default group Member"

Without quotes

**Comment by Ask4Lp** [comment]
Thanks for the quick answer, it works now, thanks!

**Comment by CzEMC** [comment]
No problem! Glad that I can help.

Closes #2301

**Comment by zml2008** [comment]
Also it looks like you're running a *very* old version of PEX -- get the latest version from [the download page](http://dev.bukkit.org/bukkit-plugins/permissionsex/files/).
